addon/webextension/content-script.js

```diff
@@ -41,11 +47,12 @@ function findAndReplace(wordList) {
   document.querySelectorAll(".donotdelete").forEach((node) => {
     const hoverEle = document.createElement("span");
     hoverEle.innerHTML = `
```

Comment:
I'm getting this locally from ESLint:

```
/Users/pdehaan/dev/github/mozilla/addon-wr/addon/webextension/content-script.js
   49:5   error    Unsafe assignment to innerHTML                                   no-unsanitized/property
  108:10  warning  Expected a conditional expression and instead saw an assignment  no-cond-assign
  112:11  warning  Expected a conditional expression and instead saw an assignment  no-cond-assign

✖ 3 problems (1 error, 2 warnings)
```

It was an `.innerHTML` before, so doesn't look like this is a new problem. Not sure if we just want to slap an
`// eslint-disable-next-line no-unsanitized/property` on the previous line to appease the ESLint gods (since we know what we're doing and accept the risks).